examples/with-emotion-vanilla/pages/_app.js

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react'
+import { hydrate } from '@emotion/css'
+
+// Adds server generated styles to emotion cache.
+// this needs to come before the app mounts
+// We use query selector here as setting on window results in emotion error `Cannot read property 'forEach' of undefined`
+if (typeof window !== 'undefined') {
+  const ids = JSON.parse(
+    document
+      .querySelector('[data-emotion-ssr]')
+      .getAttribute('data-emotion-ssr')
+  )
+  hydrate(ids)
```

Comment:
I believe that explicit hydrating is not needed since Emotion 10

Reply:
Also - the `data-emotion` format has changed in v11 so this is definitely not 100% correct for v11.

Reply:
Cool - thanks for that. Your correct i have it working locally i will commit this in a new PR - to remove comments etc